### PR TITLE
Staging CI

### PR DESCRIPTION
**Pull request by manorit2001**
`staging` branch will remain in sync with open testing applications

### CODE REVIEW

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+name: Staging Build
+
+on:
+  push:
+    branches:
+      [staging-CI]
```
Comment (manorit2001) [inline .github/workflows/staging.yml:6]:
```suggestion
      [staging]
```

File: .github/workflows/staging.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        with:
+          fetch-depth: 0
+      - name: Get tagname of push
+        id: check
+        run: |
+          SHA=$(git log -n1 --format=format:"%H")
+          git checkout master
+          git describe $SHA
+          echo "The tag is " $(git describe $SHA)
+          echo ::set-output name=tagname::$(git describe $SHA) # change to exact-match
```
Comment (manorit2001) [inline .github/workflows/staging.yml:22]:
```suggestion
          git describe --exact-match $SHA
          echo "The tag is " $(git describe --exact-match $SHA)
          echo ::set-output name=tagname::$(git describe --exact-match $SHA)
```
Exact match will fail if the currently pushed head is not at some tag. if we want to search for the last tag before the commit then we can use `--abbrev=0`

**Review by manorit2001 — COMMENTED** [review]
@parth-07 look at the tests once, Ig only these changes need to be done after the testing is done so that this can be merged in master

**Review by parth-07 — APPROVED** [review]
Looks good.

**Pull request merged**